### PR TITLE
[DI] Guard against invalid probe config and related edge-cases

### PR DESCRIPTION
**Pull request by watson**
We have seen in the system tests that it tried to call the `ack`
callback twice. We have not been able to reproduce it or have it occur
more than once in the system tests, so we don't know what caused it. But
in any case, this updated code is safer in case it ever happens again.

### CODE REVIEW

**Comment by watson** [comment]
* **#4816** <a href="https://app.graphite.dev/github/pr/DataDog/dd-trace-js/4816?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @watson and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/dd-trace/src/debugger/index.js [hunk 1]
```diff
@@ -24,13 +25,19 @@ function start (config, rc) {
   configChannel = new MessageChannel()
 
   rc.setProductHandler('LIVE_DEBUGGING', (action, conf, id, ack) => {
-    const ackId = `${id}-${conf.version}`
-    rcAckCallbacks.set(ackId, ack)
+    rcAckCallbacks.set(++ackId, ack)
```
Comment (bengl) [inline packages/dd-trace/src/debugger/index.js:28]: In theory you'd have to worry about running into MAX_SAFE_INTEGER, but I think even this happens once per milisecond, you'll have centuries of leeway.

**Review by bengl — APPROVED** [review]
